src/common/helpers/metrics.js

```diff
@@ -25,4 +25,17 @@ const metricsCounter = async (metricName, value = 1) => {
   }
 }
 
-export { metricsCounter }
+const logWarningMetrics = async (warningCount, endpointType) => {
+  await metricsCounter(`warningsReturned${endpointType}`, warningCount)
```

Comment:
is it possible to set tags in a single warningsReturned metric instead of building a metric with string concatenation?